reana_job_controller/utils.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of REANA.
+# Copyright (C) 2017, 2018, 2019 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA-Job-Controller utils."""
+
+import paramiko
```

Comment:
CC @khurtado with whom we discussed a few days ago which Python library to use for emulating SSH-to-Headnode technique for Slurm. Have you had in mind `paramiko`?

Reply:
@tiborsimko Yes, `paramiko` is what I had in mind. Perfect for SSH, just not compatible with GSI-SSH, which is relevant for e.g.: HPC centers, but shouldn't be an issue for the CERN case.